src/table/utils/handle-accessibility.js

```diff
@@ -58,7 +59,20 @@ export const handleResetFocus = ({
     const focusType = shouldRefocus.current ? 'focus' : 'addTab';
     shouldRefocus.current = false;
     const rowElements = rootElement.getElementsByClassName('sn-table-row');
-    updateFocus({ focusType, rowElements, cellCoord: nextCell });
+    // updateFocus({ focusType, rowElements, cellCoord: nextCell });
+
+    const providedCell = rowElements[nextCell[0]]?.getElementsByClassName('sn-table-cell')[nextCell[1]];
+    updateFocus({ focusType, providedCell });
+
+    const hasActiveClassName = providedCell?.classList?.contains('selected');
```

Comment:
Move this inside the if block. And name it `hasSelectedClassName`.

Reply:
✅.  Done